pixi.toml

```diff
@@ -144,6 +145,22 @@ _install-ui = 'playwright install chromium'
 cmd = 'pytest panel/tests/ui --ui --browser chromium -n logical --dist loadgroup --reruns 3 --reruns-delay 10'
 depends_on = ["_install-ui"]
 
+# =============================================
+# ================== TYPES ====================
+# =============================================
+[feature.type.dependencies]
+mypy = "*"
+pandas-stubs = "*"
```

Comment:
`typing-extensions` usually useful to have for features that may have been backported from a newer version of Python

Reply:
Agreed, just copied the dependencies over from pyproject.

Reply:
I've added typing-extensions.